eldritch/encounters.py

```diff
@@ -151,17 +151,119 @@ def Asylum1(char):
   roll3 = events.Gain(char, {"clues": 3})
   cond = events.Conditional(char, check, "successes", {0: roll0, 1: roll1, 3: roll3})
   return events.Sequence([check, cond], char)
+def Asylum2(char):
+  check = events.Check(char, "speed", -1)
+  item = events.Draw(char, "unique", 1)
+  move = events.ForceMovement(char, "Downtown")
+  cond = events.Conditional(char, check, "successes", {0: move, 1: item})
+  return events.PassFail(char, check, item, move)
+def Asylum3(char):
+  check = events.Check(char, "sneak", -1)
+  escape = events.ForceMovement(char, "Downtown")
+  arrested = events.Arrested(char)
+  return events.PassFail(char, check, escape, arrested)
+def Asylum4(char):
+  check = events.Check(char, "lore", -1)
+  spell = events.Draw(char, "spells", 1)
+  return events.PassFail(char, check, spell, events.Nothing())
+def Asylum5(char):
+  check = events.Check(char, "will", -1)
+  lose = events.Nothing() #TODO: Oh dear...so many choices
+  skill = events.Draw(char, "skills", 1)
+  cond = events.Conditional(char, check, "successes", {0: lose, 2: skill})
+  return events.Sequence([check, cond], char)
+def Asylum6(char):
+  check = events.Check(char, "lore", -2)
+  gain = events.Gain(char, {"clues": 2})
+  loss = events.Loss(char, {"stamina": 1})
+  return events.PassFail(char, check, gain, loss)
+def Asylum7(char):
+  check = events.Check(char, "fight", -2)
+  stamina = events.Gain(char, {"stamina": 2})
+  rest = events.Sequence([stamina, events.LoseTurn(char)], char)
+  fight = events.PassFail(char, check, events.Nothing(), rest)
+  return events.BinaryChoice(char, "Do you resist?", "Yes", "No", fight, rest)
 
 def Bank1(char):
-    return events.Nothing() # TODO: implement location choice
+  return events.Nothing() # TODO: implement location choice
+def Bank2(char):
+  check = events.Check(char, "luck", -1)
+  spend = events.Loss(char, {"dollars": 2})
+  common = events.Draw(char, "common", 1)
+  unique = events.Draw(char, "unique", 1)
+  cond = events.Conditional(char, check, "successes", {0: common, 1: unique})
+  prereq = events.AttributePrerequisite(char, "dollars", 2, "at least")
+  nothing = events.Nothing()
+  choice = events.BinaryChoice(char, "Pay $2 for man's last possession?",
```

Comment:
When the arg list is very long, I prefer to have nothing on the line with the open paren, and just start a new line.

Reply:
done